### PR TITLE
Design Hack: Remove icons from chart titles

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue
The icons take up too much space on mobile.

## Description

Removes the icons from the chart titles but keep the icon files themselves if we want to use them in the future.

### Double check

- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: web/src/features/charts/ChartTitle.tsx [hunk 1]
```diff
@@ -8,11 +8,10 @@ type Props = {
   translationKey: string;
   hasLink?: boolean;
   badgeText?: string;
-  icon?: JSX.Element;
   unit?: string;
 };
 
-export function ChartTitle({ translationKey, badgeText = undefined, icon, unit }: Props) {
```
Comment (silkeholmebonnen) [inline web/src/features/charts/ChartTitle.tsx:15]: If we keep the icon files, should we keep this prop as well? Or do you mean that we might use the icons elsewhere in the future?

Comment (VIKTORVAV99) [reply web/src/features/charts/ChartTitle.tsx:15]: We might use them elsewhere in the future. I don't think we will be using them inside the titles though  so it's fine to remove the prop but @Alportan can confirm.

**Review by silkeholmebonnen — APPROVED** [review]
LGTM 🎉 

I believe Alex told me the same thing about the icons in the title

File: web/src/features/charts/ChartTitle.tsx (hunk 1 above)
Comment (Alportan) [reply web/src/features/charts/ChartTitle.tsx:15]: We won't be using them inside titles so all good 👌

**Pull request merged**